intelmq/bots/parsers/shadowserver/_config.py

```diff
@@ -3040,8 +3543,7 @@ def scan_exchange_identifier(field):
     ('Outdated-DNSSEC-Key', 'outdated_dnssec_key', outdated_dnssec_key),
     ('Outdated-DNSSEC-Key-IPv6', 'outdated_dnssec_key_v6', outdated_dnssec_key),
     ('SSL-FREAK-Vulnerable-Servers', 'scan_ssl_freak', ssl_freak_vulnerable_servers),
-    ('SSL-POODLE-Vulnerable-Servers IPv4', 'scan_ssl_poodle', ssl_poodle46_vulnerable_servers),
-    ('SSL-POODLE-Vulnerable-Servers IPv6', 'scan6_ssl_poodle', ssl_poodle46_vulnerable_servers),
+    ('SSL-POODLE-Vulnerable-Servers', 'scan_ssl_poodle', ssl_poodle_vulnerable_servers),
```

Comment:
Why did you remove the IPv4/IPv4 file name mappings here? Did shadowserver stop sending reports per IP protocol and just one for both v4 and v6?